tailwind.config.ts

```diff
@@ -3,25 +3,34 @@
 
 import fs from "fs";
 import path from "path";
+import formsPlugin from "@tailwindcss/forms";
 import plugin from "tailwindcss/plugin";
 
-module.exports = {
+export default {
   content: [
     "./js/**/*.js",
     "./js/**/*.ts",
     "./js/**/*.tsx",
     "./lib/orbit_web.ex",
     "./lib/orbit_web/**/*.*ex",
+    "./storybook/**/*.html",
   ],
+  darkMode: ["class", '[data-mode="dark"]'],
   theme: {
-    extend: {
-      colors: {
-        brand: "#FD4F00",
+    colors: {
+      "mbta-red": "#DA291C",
+      "mbta-orange": "#ED8B00",
+      "mbta-blue": "#003DA5",
+      black: "#000000",
+      gray: {
+        100: "#EAEAEA",
+        200: "#D9D9D9",
+        300: "#929292",
```

Comment:
Where do these come from?
Tailwindcss has builtin grays, but they don't line up with these: https://tailwindcss.com/docs/customizing-colors

Reply:
From the Figma designs linked in the Asana ticket.